cmd/juju/commands/run.go

```diff
@@ -53,7 +53,7 @@ are able to use this command.
 Targets are specified using either machine ids, application names or unit
 names.  At least one target specifier is needed.
 
-Multiple values can be set for --machine, --application, and --unit by using
+Multiple values can be set for --machine, --application/--app/-a, and --unit/-u by using
```

Comment:
I wouldn't complicate the comment here. Keep all this as --application --unit etc
Just add a new, standalone paragraph which says that --application can be shorted to --app or -a, --unit to -u etc